dts/arm/ti/j721e-pinctrl.dtsi

```diff
@@ -0,0 +1,25 @@
+/* Copyright (C) 2023 BeagleBoard.org Foundation
+ * Copyright (C) 2023 S Prashanth
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/dt-bindings/pinctrl/ti-k3-pinctrl.h>
+
+&pinctrl {
+	uart0_tx_default: uart0_tx_default {
+		pinmux = <0x1e8 (PIN_OUTPUT | 0)>;
+	};
+
+	uart0_rx_default: uart0_rx_default {
```

Comment:
I don't see ``uart0_*_default`` being used anywhere else, what is the purpose of adding it then?

Reply:
removing it.